packages/canary-integration/src/__tests__/model-instance-document.test.ts

```diff
@@ -52,33 +74,50 @@ describe('ModelInstanceDocument API http-client tests', () => {
     await ipfs.stop()
   })
 
-  test('Create valid doc', async () => {
-    const doc = await ModelInstanceDocument.create(ceramic, CONTENT0, METADATA)
+  test('verifies the content against model schema', async () => {
```

Comment:
might be good to have a second test for updating the doc to an invalid schema - so we have both the genesis commit and applying a signed commit paths tested